content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -87,16 +87,16 @@ The three conditions are `ready`, `serving`, and `terminating`.
 
 #### Ready
 
-`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
-condition set to `True` should have this EndpointSlice condition also set to `true`. For
+`ready` is a condition that maps to a Pod's `ready` condition. A running Pod with the `ready`
```

Comment:
No. Pod's condition is called `Ready`, not `ready`.

Reply:
@YashPimple Please do the changes according to the suggestion.